app/scripts/modules/core/src/pipeline/config/stages/evaluateVariables/evaluateVariablesStage.ts

```diff
@@ -0,0 +1,16 @@
+import { Registry } from 'core/registry';
+
+import { ExecutionDetailsTasks } from '../core';
+import { EvaluateVariablesExecutionDetails } from './EvaluateVariablesExecutionDetails';
+import { EvaluateVariablesStageConfig } from './EvaluateVariablesStageConfig';
+
+Registry.pipeline.registerStage({
+  label: 'Evaluate Variables',
+  description: 'evalutes variables',
```

Comment:
Can we change to "Evaluates variables for use in SpEL expressions in downstream stages. Variables can be accessed by their key."?